src/main/java/com/github/onsdigital/logging/builder/LogMessageBuilder.java

```diff
@@ -79,38 +65,36 @@ public DateFormat getDataFormat() {
     }
 
     public LogMessageBuilder addParameter(String key, Object value) {
-        this.parameters.getParameters().put(key, value);
+        parameters.getParameters().put(key, value);
         return this;
     }
 
+    public void setNamespace(String namespace) {
+        if (StringUtils.isNotEmpty(namespace)) {
```

Comment:
Should we be throwing an exception here if `namespace` is not set?

Reply:
I don't think so. I don't think the app should stop working if logging isn't right, plus there is already a package field which will be something like `com.github.onsdigital.thetrain`. Also need to be careful not to break backwards compatibility.